software/slskd.yml

```
@@ -0,0 +1,11 @@
+name: "slskd"
+website_url: "https://github.com/slskd/slskd"
+source_code_url: "https://github.com/slskd/slskd"
+description: "A modern client-server application for the Soulseek file sharing network."
+licenses:
+  - AGPL-3.0
+platforms:
+  - C#
```

Add Docker accordingly to your installation documentation
```suggestion
platforms:
  - Docker
  - C#
```